wagtail/project_template/home/templates/home/welcome_page.html

```diff
@@ -0,0 +1,377 @@
+{% templatetag openblock %} load i18n {% templatetag closeblock %}
+<link rel="stylesheet" type="text/css" href="/static/admin/css/fonts.css">
+<style type="text/css">
+  body, main {
+    margin: 0 auto;
+  }
+  .body, .tip {
+    stroke: #fff;
+  }
+  html {
+    line-height: 1.15;
+    box-sizing: border-box;
+  }
+  footer, header, main {
+    display: block;
+  }
+  a {
+    background-color: transparent;
+    -webkit-text-decoration-skip: objects;
+    color: #246060;
+    text-decoration: none;
+  }
+  a:hover {
+    text-decoration: underline;
+  }
+  img {
+    border-style: none;
+  }
+  header {
+    border-bottom: 1px solid #efefef;
+  }
+  svg:not(:root) {
+    overflow: hidden;
+  }
+  body {
+    max-width: 960px;
+    color: #000000;
+    font-family: Open Sans, Arial, sans-serif;
+  }
+  main {
+    text-align: center;
+  }
+  h1, h2, h3, h4, h5, p, ul {
+    padding: 0;
+    margin: 0;
+    font-weight: 400;
+  }
+  header {
+    padding-top: 20px;
+    padding-bottom: 10px;
+  }
+  .row {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  .logo {
+    width: 200px;
+  }
+  .release-notes {
+    font-size: 14px;
+  }
+  .figure {
+    margin-top: 19vh;
+    height: 173px;
+  }
+  .figure-space {
+    max-width: 265px;
+    position: relative;
+    z-index: -9;
+  }
+  .figure-logo {
+    max-width: 150px;
+    position: relative;
+    z-index: -9;
+  }
+  .body {
+    fill: #6DDCBD;
+    -webkit-transform: scaleX(1.25);
+    transform: scaleX(1.25);
+    -webkit-transform-origin: center;
+    transform-origin: center;
+    stroke-width: 3px;
+  }
+  .box {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+  }
+  .circle {
+    fill: #6DDCBD;
+    stroke: #fff;
+    stroke-width: 7px;
+  }
+  h2 {
+    font-size: 22px;
+    max-width: 500px;
+    margin: 5px auto 0;
+  }
+  main p {
+    font-size: 16px;
+    line-height: 20px;
+    max-width: 390px;
+    margin: 15px auto 0;
+    color: #000000;
+  }
+  footer {
+    padding: 25px 0;
+    position: fixed;
+    left: 50%;
+    bottom: 0;
+    width: 960px;
+    -webkit-transform: translateX(-50%);
+    transform: translateX(-50%);
+    -webkit-transform-style: preserve-3d;
+    transform-style: preserve-3d;
+    border-top: 1px solid #efefef;
+  }
```

Comment:
Could you rewrite this with Flexbox-style [sticky footer](https://philipwalton.github.io/solved-by-flexbox/) instead? This will be easier to keep this working if some day the page includes more content and starts to scroll on some screen sizes.

Reply:
Fixed in 959d41b. I elected to not make it sticky, as it seemed to add unnecessary complexity, and the height of the main content area is short enough that it creates a rather large, odd-looking gap on tall windows. If anyone disagrees with this, happy to switch it back. The technique Thibaud linked to is very straightforward.